vendor/sendgrid-php/sendgrid-php.php

```diff
@@ -1,4 +1,4 @@
 <?php
-require 'vendor/autoload.php';
-require 'lib/SendGrid.php';
-?>
\ No newline at end of file
+require __DIR__ . '/vendor/autoload.php';
+require __DIR__ . '/lib/SendGrid.php';
```

Comment:
Instead of / you should use DIRECTORY_SEPARATOR php constant. Ex:

require __DIR__ . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'SendGrid.php';